<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>239</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>239</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>239</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Choose a Connection</string>
  </property>
  <property name="windowIcon" >
   <iconset>icons/prod_color.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headerFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>500</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>500</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="headerLeft" >
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap>images/banner_left_short.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="headerRight" >
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap>images/banner_right_short.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="mainFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>207</green>
           <blue>210</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>8</number>
      </property>
      <property name="leftMargin" >
       <number>10</number>
      </property>
      <property name="topMargin" >
       <number>10</number>
      </property>
      <property name="rightMargin" >
       <number>10</number>
      </property>
      <property name="bottomMargin" >
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="outerCosmeticFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>127</red>
              <green>157</green>
              <blue>185</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>127</red>
              <green>157</green>
              <blue>185</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>118</red>
              <green>116</green>
              <blue>108</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="leftMargin" >
          <number>10</number>
         </property>
         <property name="topMargin" >
          <number>10</number>
         </property>
         <property name="rightMargin" >
          <number>10</number>
         </property>
         <property name="bottomMargin" >
          <number>10</number>
         </property>
         <item>
          <widget class="QFrame" name="innerCosmeticFrame" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>208</red>
                 <green>208</green>
                 <blue>191</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>234</green>
                 <blue>223</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>208</red>
                 <green>208</green>
                 <blue>191</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>234</green>
                 <blue>223</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>118</red>
                 <green>116</green>
                 <blue>108</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>234</green>
                 <blue>223</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>234</green>
                 <blue>223</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground" >
            <bool>true</bool>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>12</number>
            </property>
            <property name="leftMargin" >
             <number>10</number>
            </property>
            <property name="topMargin" >
             <number>10</number>
            </property>
            <property name="rightMargin" >
             <number>10</number>
            </property>
            <property name="bottomMargin" >
             <number>10</number>
            </property>
            <item>
             <widget class="QFrame" name="labelFrame" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <property name="leftMargin" >
                <number>10</number>
               </property>
               <property name="topMargin" >
                <number>0</number>
               </property>
               <property name="rightMargin" >
                <number>10</number>
               </property>
               <property name="bottomMargin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelDialogMsg" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>0</width>
                   <height>34</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>16777215</width>
                   <height>34</height>
                  </size>
                 </property>
                 <property name="palette" >
                  <palette>
                   <active>
                    <colorrole role="WindowText" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>0</red>
                       <green>0</green>
                       <blue>0</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </active>
                   <inactive>
                    <colorrole role="WindowText" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>0</red>
                       <green>0</green>
                       <blue>0</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </inactive>
                   <disabled>
                    <colorrole role="WindowText" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>127</red>
                       <green>127</green>
                       <blue>127</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </disabled>
                  </palette>
                 </property>
                 <property name="font" >
                  <font>
                   <family>Arial</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text" >
                  <string>More than one connection profile is configured for this network.  Please choose the one you wish to use to connect</string>
                 </property>
                 <property name="alignment" >
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap" >
                  <bool>true</bool>
                 </property>
                 <property name="indent" >
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" >
               <property name="spacing" >
                <number>6</number>
               </property>
               <property name="leftMargin" >
                <number>0</number>
               </property>
               <property name="topMargin" >
                <number>0</number>
               </property>
               <property name="rightMargin" >
                <number>0</number>
               </property>
               <property name="bottomMargin" >
                <number>0</number>
               </property>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>110</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxConnection" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>16777215</width>
                   <height>25</height>
                  </size>
                 </property>
                 <item>
                  <property name="text" >
                   <string>40-bit Hex</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>104-bit Hex</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>110</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox" >
        <property name="standardButtons" >
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
